File: 2274-keep-multiplying-found-values-by-two/2274-keep-multiplying-found-values-by-two.js
/**
 * @param {number[]} nums
 * @param {number} original
 * @return {number}
 */
var findFinalValue = function(nums, original) {
   let sum = 0
   for(let i=0; i<nums.length; i++){
    if(nums[i] === original){
        sum = nums[i]*2
        original = sum
        i = -1
    }else{
        sum = original
    }
   }
   return sum
 
   
};

File: 2274-keep-multiplying-found-values-by-two/2274-keep-multiplying-found-values-by-two.cs
public class Solution {
    public int FindFinalValue(int[] nums, int original) {
        int i = 0;
do
{
    if (nums[i] == original)
    {
        original = nums[i] * 2;
        i=-1;
        Console.WriteLine(original);
    }
      i++;

}
while (i < nums.Length);
return original;
    }
}